infra/test-core2wls.rkt

```diff
@@ -13,8 +13,7 @@
     (lambda (port)
       (fprintf port "~a\n" (*prog*))
       (fprintf port
-        (format "Block[{$MinPrecision=~a,$MaxPrecision=~a,$MaxExtraPrecision=0},TimeConstrained[MemoryConstrained[Print[f[~a]//N],2^32],5]]\n"
-                wls-prec wls-prec
+        (format "TimeConstrained[MemoryConstrained[Print[f[~a]//N],2^32],5]\n"
```

Comment:
I think this is fine in this PR, but IIRC Mathematica distinguishes between "real" numbers and "limited-precision" numbers, and I'm not sure this actually preserves the semantics correctly. Let's leave that as a separate issue, though.

Reply:
I might revert this, I'm unsure of what Mathematica actually is doing with all of those functions.

Reply:
Ok reverted. Either way, Mathematica fails tests with 150 ulp error. I'm not sure if Mathematica can be convinced to do the "right" thing.